### PR TITLE
[KAN-30] communicate FE and BE

### PR DESCRIPTION
**Pull request by sgfn**

### CODE REVIEW

File: backend/src/main/java/com/rikishi/rikishi/model/Fight.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    User firstUser,
+    User secondUser,
+    int number,
+    WeightClass weightClass,
+    // -1 if no winner. Yes I know it's an anti-pattern, don't have time to correct it
+    long winnerId
+) implements Indexable<Long> {
+    public static Fight fromJson(
+        Duel duel,
+        User resolvedFirstUser,
```
Comment (Endrewzz) [inline backend/src/main/java/com/rikishi/rikishi/model/Fight.java:16]: I am not sure, what does exactly resolvedFirstUser and resolvedSecondUser does store. Is this just two contestants in the fight?

**Review by Endrewzz — APPROVED** [review]
LGTM, but I have a question: what does now `long winner` store in `Duel.java`? Number which contestant won (1 or 2)?

File: backend/src/main/java/com/rikishi/rikishi/model/Fight.java (hunk 1 above)
Comment (sgfn) [reply backend/src/main/java/com/rikishi/rikishi/model/Fight.java:16]: Yeah, just that

**Comment by sgfn** [comment]
>what does now long winner store in Duel.java?

As far as I can see in the FE code, it's the ID of the winner. Updated the field name to reflect that

**Pull request merged**